demos/simple-storage/protocol/web3api.yaml

```diff
@@ -1,8 +1,9 @@
 description: SimpleStorage Web3API Example
+version: "1.0.0"
 repository: https://github.com/web3-api/prototype
 mutation:
   schema:
-    file: ./src/mutation/schema.graphql
+    file: ./src/mutation/index.ts
```

Comment:
Why are you changing this here? This is incorrect. Were you doing this to test the validation logic? Please create tests & test projects for this purpose.

Reply:
That's correct, was just for testing validation logic - Since this is a WIP PR I was testing with the already existing project. If you think we should create a test project for this no worries I will create it